sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/AlterTableRenamePartitionSuite.scala

```diff
@@ -17,13 +17,16 @@
 
 package org.apache.spark.sql.execution.command.v2
 
+import org.scalatest.Ignore
+
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.execution.command
 
 /**
  * The class contains tests for the `ALTER TABLE .. RENAME PARTITION` command
  * to check V2 table catalogs.
  */
+@Ignore
```

Comment:
Although GA passed, I am not sure we really need ignore this case ...

Reply:
Let's ignore it for now.